### ISSUE TITLE
Library doesn't install, and causes error on load

### ISSUE THREAD

**Issue by GShotwell** [dancormier/react-native-swipeout#151]
I'm getting the following errors: 

```
Unhandled JS Exception: Seems you're trying to access 'ReactNative.createClass' from the 'react-native' package. Perhaps you meant to access 'React.createClass' from the 'react' package instead?

For example, instead of:

  import React, { Component, View } from 'react-native';

You should now do:

  import React, { Component } from 'react';
  import { View } from 'react-native';

Check the release notes on how to upgrade your code - https://github.com/facebook/react-native/releases/tag/v0.25.1
```

Is this Library still being maintained?

**Comment by JUnderwood1991** [comment]
I'm having a similar issue (it just happens to show in Xcode instead) I guess this project has been abandoned...

**Comment by JUnderwood1991** [comment]
https://github.com/silentcloud/react-native-swipeout

This is a republish of the project you have, with the issues allegedly fixed.

**Comment by abumostafa** [comment]
i have been into this issue for one day then i figure it out.
the right library is  [https://www.npmjs.com/package/react-native-swipe-out](this one)
`npm install react-native-swipe-out`

**Comment by silentcloud** [comment]
I have already published version to `2.0.13`.
